ui/app/templates/components/database-connection.hbs

```diff
@@ -172,9 +171,7 @@
     <div class="field is-grouped is-grouped-split is-fullwidth box is-bottomless">
       <div class="field is-grouped">
         <div class="control">
-          <button data-test-secret-save type="submit" disabled={{this.buttonDisabled}} class="button is-primary">
```

Comment:
I didnt see anywhere where this.buttonDisabled is set, and disabling submit is an antipattern now so I just removed it